Content.Shared/PowerCell/SharedPowerCellSystem.cs

```diff
@@ -48,15 +49,15 @@ private void OnCellInserted(EntityUid uid, PowerCellSlotComponent component, Ent
 
         if (args.Container.ID != component.CellSlotId)
             return;
-
+        _appearance.SetData(uid, PowerCellSlotVisuals.Enabled, true);
         RaiseLocalEvent(uid, new PowerCellChangedEvent(false), false);
     }
 
     protected virtual void OnCellRemoved(EntityUid uid, PowerCellSlotComponent component, EntRemovedFromContainerMessage args)
     {
         if (args.Container.ID != component.CellSlotId)
             return;
-
+        _appearance.SetData(uid, PowerCellSlotVisuals.Enabled, false);
```

Comment:
does this also work for when a cell has 0 charge.

Reply:
The commit bellow should have resolved that, emo even helped me with it